compiler/luci/pass/src/ConvertNCHWToNHWCPass.cpp

```diff
@@ -316,6 +366,43 @@ class ConvertNCHWToNHWC final : public luci::CircleNodeMutableVisitor<bool>
     post_trans->a(node);
     return true;
   }
+
+  bool visit(luci::CircleMul *node)
+  {
+    LOGGER(l);
+
+    luci::CircleNode *pred_node = nullptr;
+    luci::CircleConst *gamma = nullptr;
```

Comment:
Just curious, what use name `gamma`?

Reply:
In https://en.wikipedia.org/wiki/Multiplication, there's `mutiplicand` and `multiplier`

Reply:
The name `gamma` originated from normalization operators.

This PR only supports a special form of MUL which might come from normalization Ops, so I used `gamma` instead of more general terms.

> In https://en.wikipedia.org/wiki/Multiplication, there's mutiplicand and multiplier

multiplier looks ok. I've changed gamma to multiplier.